tests/testthat/test-markdown.R

```diff
@@ -1,3 +1,10 @@
 test_that("empty string works", {
   expect_equal(markdown_text(""), "")
 })
+
+test_that("header attributes are parsed", {
+  index_xml <- markdown(normalizePath(test_path("assets/readme-with-attr.md")))
```

Comment:
Can you please use `markdown_text()` here?

Reply:
Yup!